src/renderer/features/operation/sign/ui/VaultSigning/VaultSigning.tsx

```diff
@@ -41,7 +42,10 @@ export const VaultSigning = ({
 
   const handleSignature = async (data: string | string[]): Promise<void> => {
     const isMultishard = Array.isArray(data);
-    const signatures = isMultishard ? (data as HexString[]) : [data as HexString];
+    let signatures = isMultishard ? (data as HexString[]) : [data as HexString];
```

Comment:
What about this one? Because we actually don't want to modify `signatures` in the following code
```suggestion
    const signatures = isMultishard ? (data as HexString[]).map(transformEcdsaSignature) : [data as HexString].map(transformEcdsaSignature);
```